app/gui/suggestion-database/src/entry.rs

```diff
@@ -973,10 +973,11 @@ where
                 let label = chain_to_label(&chain);
                 let qualified_name = entry.qualified_name();
                 let required_import = Some(qualified_name.to_string());
-                // As we don't generate `this`, we will never follow the code path where
-                // `in_module` is used. That's why passing `default()` is valid.
+                // Passing some meaningful module name instead of `default()` is preferable, but we
+                // can't access it in this method. This means we never use `Main` instead of
+                // project name in dropdowns.
```

Comment:
We use `Main` instead of project name, because it is more resilient to project renaming. No chance of propagating current module from somewhere?

Reply:
It was a bit painful, but done.